src/modules/esl-nested-accordion/core/esl-nested-accordion.ts

```diff
@@ -0,0 +1,27 @@
+import {ExportNs} from '../../esl-utils/environment/export-ns';
+import {ESLMixinElement} from '../../esl-mixin-element/core';
+import {ESLPanel} from '../../esl-panel/core/esl-panel';
+import {listen} from '../../esl-utils/decorators';
+
+/**
+ * ESLNestedAccordion mixin element
+ * @author Anastasia Lesun, Natalia Smirnova
+ *
+ * ESLNestedAccordion is a custom mixin element that can be used with {@link ESLPanel} instance to allow hide all children panels
+ */
+@ExportNs('NestedAccordion')
+export class ESLNestedAccordion extends ESLMixinElement {
+  static is = 'esl-nested-accordion';
+
+  /** @returns all nested panels  */
+  public get $nestedPanels(): ESLPanel[] {
```

Comment:
I don't think collecting all the panels inside is a good idea.
At least we never previously in ESL restrict users to changing the target.
We need to think about `ESLTraversingQuery` usage here instead